usd/src/sdf_parser/Link_Sdf2Usd_TEST.cc

```diff
@@ -97,6 +97,7 @@ TEST_F(UsdStageFixture, Link)
 
   std::string boxPath = worldPath + "/" + "box";
   auto box = this->stage->GetPrimAtPath(pxr::SdfPath(boxPath));
+  EXPECT_TRUE(box.HasAPI<pxr::UsdPhysicsRigidBodyAPI>());
```

Comment:
can a check be added above to ensure that the [ground plane prim](https://github.com/ignitionrobotics/sdformat/blob/158cd8333dac98a8b1d2f7a9c16c2b6f14c4b9fe/usd/src/sdf_parser/Link_Sdf2Usd_TEST.cc#L83) does not have a rigid body applied to it, since the ground plane is static?